### ISSUE TITLE
Please support --defaults-group-suffix

### ISSUE THREAD

**Issue by derobert** [dbcli/mycli#95]
With the stock MySQL command line client, you can add `--defaults-group-suffix=SOMETHING` to look for the `[mysqlSOMETHING]` section in `my.cnf`. This lets you store settings for, e.g., multiple database servers.

It'd be nice if mycli had this option as well.

**Comment by amjith** [comment]
Yes. This is a good request. I'll add this in.

**Comment by j-bennet** [comment]
@amjith: If a setting is specified in both .pgclirc and in a suffixed group of my.cnf file (such as "prompt"), which one should take priority? I'm leaning towards .pgclirc, but not sure.

**Comment by j-bennet** [comment]
Actually there's another consideration here: since we have a default .myclirc, even if user erases the setting from his "home" version of .myclirc, we're going to read it from default config. If I prioritize it over "suffixed group" setting, as was my initial thought, it may bee to much pain for a user to force mycli into using the "suffixed group" setting. He'd have to erase the settings from both "home config" and "default config", and only then "suffixed group" setting would be used. Hmm. :-/

**Comment by amjith** [comment]
Always give higher precedence to my.cnf.

**Comment by amjith** [comment]
This is now available in master. You can try it out by installing it directly from master as follows: 

```
pip install https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by amjith** [comment]
This is now available in the latest version of mycli (1.3.0). Please upgrade.

**Comment by derobert** [comment]
Thank you. Confirmed working here.
